### PR TITLE
add @Kiwi to trusted users

### PR DESCRIPTION
**Pull request by Kiwi**
I'm trying to get more active contributing to nixpkgs so making myself known to ofborg seems like a good idea and also I'd like to add support for macOS to a few packages I already maintain sometime real soon (tm) so trusted users it is.

I'm like really trustworthy, I've had access to the community aarch64 server for months and only broke it once! Just look at my face :smirk: (jk, I haven't broken it... yet) Oh, and also @grahamc said to, so there's that. :smile:

### CODE REVIEW

File: config.public.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
             "jonringer",
             "jtojnar",
             "kalbasit",
+	    "kiwi",
```
Comment (ryantm) [inline config.public.json]: Seems like your indentation is different.

**Review by ryantm — CHANGES_REQUESTED** [review]

File: config.public.json (hunk 1 above)
Comment (Kiwi) [reply config.public.json]: Somehow a \t snuck in... I never use tabs. Weird.

**Review by ryantm — APPROVED** [review]

**Review by worldofpeace — APPROVED** [review]

**Comment by worldofpeace** [comment]
:eyes:

**Comment by Kiwi** [comment]
It only took me a little over a year but I now have a macOS builder and have been updating and testing as much as I can on aarch64-linux, x86_64-linux, and x86_64-darwin! :smile: I still don't know how to use the bot. Anyway, I should have them all done about the same time as nixpkgs supports the new apple arm machines or at least soon after... so I'll check back in same time next year and let you know how it's going. :smiley:

**Pull request merged**